package.json

```diff
@@ -54,10 +54,11 @@
     "stylelint-scss": "^4.1.0"
   },
   "devDependencies": {
+    "@skyscanner/stylelint-plugin-backpack": "^1.2.0",
```

Comment:
This should be in `dependencies` instead of `devDependencies `.
Raised this pr to fix it: https://github.com/Skyscanner/stylelint-config-skyscanner/pull/183